acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java

```diff
@@ -69,6 +72,7 @@ public class Signer extends FilecoinJsonRpcEndpoint {
   private final Vertx vertx;
   private final String urlFormatting;
   private final Optional<ClientTlsConfig> clientTlsConfig;
+  private JsonRpc2_0Web3j signerJsonRpc;
```

Comment:
nit: not sure why we named this jsonRPC instance signer if there isn't nothing signer specifically about it

Reply:
yes not sure either. I think copied some code that used that name. but doesn't make sense. renamed it to jsonRpc